app/services/container_dashboard_service.rb

```diff
@@ -49,6 +51,13 @@ def image_metrics_data
     }.compact
   end
 
+  def refresh_status_data
+    {
+      :default_cred => @ems.present? && @ems.enabled ? 'Valid' : 'Invalid',
```

Comment:
Can you add parenthesis here to show the intended order of operations?  I know what the answer is, but it could be readable as either of the following and it's confusing otherwise:

```
(A && B) ? X : Y
A && (B ? X : Y)
```

Reply:
having parentheses giving a linter error.

== app/services/container_dashboard_service.rb ==
C: 55: 24: Style/TernaryParentheses: Omit parentheses for ternary conditions.

Reply:
ok, I disagree with rubocop here, but let's keep it removed.

Reply:
Yeah if there are multiple boolean conditions I think parens are a good idea, I don't want to have to think about operator precedence when a simple `(A && B) ? X : Y` makes it obvious what is intended.